server/test/render.test.js

```diff
@@ -860,7 +868,8 @@ describe('render', () => {
 	});
 
 	describe('state locking', () => {
-		it('should set _dirty to true', () => {
+		// TODO: we use flags now
+		it.skip('should set _dirty to true', () => {
```

Comment:
Skipped for now due to usage of flags

Reply:
Could we open an issue to track updating this or removing it if we determine it no longer makes sense?

Reply:
Will do

Reply:
Opened #3859